src/main/java/seedu/club/logic/commands/FindCommand.java

```diff
@@ -1,23 +1,42 @@
 package seedu.club.logic.commands;
 
-import seedu.club.model.member.NameContainsKeywordsPredicate;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_GROUP;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_MATRIC_NUMBER;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.club.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.club.model.member.FieldContainsKeywordsPredicate;
 
 /**
- * Finds and lists all members in club book whose name contains any of the argument keywords.
- * Keyword matching is case sensitive.
+ * Finds and lists all members in club book whose field contains any of the argument keywords.
+ * Partial match is acceptable.
+ * Keyword matching is case insensitive.
+ * Finds by all possible fields if prefix is not specified
  */
 public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all members whose names contain any of "
-            + "the specified keywords (case-sensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
-
-    private final NameContainsKeywordsPredicate predicate;
-
-    public FindCommand(NameContainsKeywordsPredicate predicate) {
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all members whose field contain any of "
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
+            + "Insert field prefix after find to search by field, otherwise find will search all fields.\n"
```

Comment:
I think it'll be clearer for the user if you mark the word 'find' like this - `find`